### DISCUSSION TITLE
How can I run validation on a different worker or node than training?

### DISCUSSION THREAD

**Discussion by jlu-figma** [Lightning-AI/pytorch-lightning#20613] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
My validation set is large and my model pauses all progress every N steps to run validation. This is slowing down my training a lot.
